pkg/operator/controller/ingress/dns_test.go

```diff
@@ -109,6 +109,70 @@ func TestDesiredWildcardDNSRecord(t *testing.T) {
 	}
 }
 
+func TestDomainNotMatching(t *testing.T) {
+	tests := []struct {
+		name         string
+		domain       string
+		baseDomain   string
+		platformType configv1.PlatformType
+		expected     bool
+	}{
+		{
+			name:         "empty domain",
+			domain:       "",
+			baseDomain:   "openshift.example.com",
+			platformType: configv1.AWSPlatformType,
+			expected:     false,
```

Comment:
I would expect us not to manage DNS if the domain is empty, so it would make sense to me to consider the empty domain always to be a non-match.

Reply:
That sounds reasonable. I've applied it.